test/performance/mako/sidecar.go

```diff
@@ -16,9 +16,47 @@ limitations under the License.
 
 package mako
 
+import (
+	"context"
+	"fmt"
+
+	"knative.dev/pkg/changeset"
+	"knative.dev/pkg/injection/clients/kubeclient"
+	"github.com/golang/protobuf/proto"
+        "github.com/google/mako/helpers/go/quickstore"
+        qpb "github.com/google/mako/helpers/proto/quickstore/quickstore_go_proto"
+)
+
 const (
 	// SidecarAddress is the address of the Mako sidecar to which we locally
 	// write results, and it authenticates and publishes them to Mako after
 	// assorted preprocessing.
-	SidecarAddress = "localhost:9813"
+	sidecarAddress = "localhost:9813"
 )
+
+// SetupMako sets up the mako client for the provided benchmarkKey.
+// It will add a few common tags and allows each benchmark to add custm tags as well.
+// It returns the mako client handle to sotre metrics, a method to close the connection
+// to mako server once done and error if case of failures.
+func SetupMako(ctx context.Context, benchmarkKey string, extraTags []string)(*quickstore.Quickstore, func(context.Context), error) {
```

Comment:
If you make extra tags variadic, then the callsites get simpler
```suggestion
func SetupMako(ctx context.Context, benchmarkKey string, extraTags ...string)(*quickstore.Quickstore, func(context.Context), error) {
```